org.eclipse.lemminx/src/test/java/org/eclipse/lemminx/uriresolver/FileServer.java

```diff
@@ -53,7 +54,7 @@ public FileServer() throws IOException {
 	public FileServer(Path baseDir) throws IOException {
 		Files.createDirectories(baseDir);
 		ResourceHandler resourceHandler = new ModifiedResourceHandler();
-		resourceHandler.setResourceBase(baseDir.toUri().toString());
+		resourceHandler.setBaseResource(new PathResource(baseDir.toRealPath()));
```

Comment:
on Mac, the tmp dir is an alias, the ResourceHandler doesn't serve aliased resources